packages/sst/src/runtime/handlers/go.ts

```diff
@@ -59,13 +59,23 @@ export const useGoHandler = Context.memo(async () => {
       sources.set(input.functionID, project);
       const src = path.relative(project, input.props.handler!);
 
+      const ldFlags = input.props.go?.ldFlags || ["-s", "-w"];
+      const buildTags = input.props.go?.buildTags || [];
+
       if (input.mode === "start") {
         try {
           const target = path.join(input.out, handlerName);
           const srcPath =
             os.platform() === "win32" ? src.replaceAll("\\", "\\\\") : src;
           const result = await execAsync(
-            `go build -ldflags "-s -w" -o "${target}" ./${srcPath}`,
+            [
+              "go",
+              "build",
+              ...(ldFlags ? [`-ldflags "${ldFlags.join(" ")}"`] : []),
+              ...(buildTags ? [`-t ${buildTags.join(" ")}`] : []),
```

Comment:
This ternary is always going to be true because you default buildTags to `|| []`. We can either not default it or check the length.